### PR TITLE
ocamlPackages.markup: 0.8.2 -> 1.0.0, ocamlPackages.lambdasoup: 0.7.1 -> 0.7.2

### PR DESCRIPTION
**Pull request by sternenseemann**
###### Motivation for this change

<https://github.com/NixOS/nixpkgs/pull/106069#pullrequestreview-545719578>

cc @vbgl (lambdasoup) @FlorentBecker (markup)

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by sternenseemann** [comment]
@Grahamcofborg build ocamlPackages.lambdasoup

File: pkgs/development/ocaml-modules/lambdasoup/default.nix [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
     owner = "aantron";
     repo = pname;
     rev = version;
-    sha256 = "14lndpsnzjjg58sdwxqpsv7kz77mnwn5658lya9jyaclj8azmaks";
+    sha256 = "0php51lyz3ll0psazjd59yw02xb9w84150gkyiwmn3fa0iq8nf7m";
   };
 
   propagatedBuildInputs = [ markup ];
 
+  doCheck = true;
```
Comment (vbgl) [inline pkgs/development/ocaml-modules/lambdasoup/default.nix:18]:
```suggestion
  doCheck = lib.versionAtLeast ocaml.version "4.04";
```

Comment (sternenseemann) [reply pkgs/development/ocaml-modules/lambdasoup/default.nix]: Thanks, fixed!

**Comment by vbgl** [comment]
Merged into `master` as 1f850a06903cb65030c5a846fb5bf7086758dabc, 2830a80142898caddf34632cd18b1c7fa2be5d1b, and 30526f59a11838a45af8031c90ebf818aa74225e.

**Pull request closed without merging**